src/clnt_vc.c

```diff
@@ -236,11 +236,11 @@ clnt_vc_ncreatef(const int fd,	/* open file descriptor */
 	ct->ct_closeit = false;
 	ct->ct_wait.tv_usec = 0;
 	ct->ct_waitset = false;
-	ct->ct_addr.buf = mem_alloc(raddr->maxlen);
 
-	memcpy(ct->ct_addr.buf, raddr->buf, raddr->len);
-	ct->ct_addr.len = raddr->len;
-	ct->ct_addr.maxlen = raddr->maxlen;
+	if (sizeof(struct sockaddr_storage) >= raddr->len) {
```

Comment:
I understand it's a bug fix, but you're not doing anything if the check fails, except silently not copying the address.  If you check for a valid address passed in, you need to handle the failure case.  Always!